include/cinder/gl/Texture.h

```diff
@@ -269,7 +278,7 @@ class TextureBase {
 		bool				mImmutableStorage;
 		GLfloat				mMaxAnisotropy;
 		GLint				mInternalFormat, mDataType;
-		GLint				mDataFormat;
+		//GLint				mDataFormat;
```

Comment:
Is this commented out because it is no longer used? If so should remove entirely.